Solutions/Menes.Abstractions/Menes/Exceptions/OutputBuilderNotFoundException.cs

```diff
@@ -4,78 +4,137 @@
             {
-                this.OpenApiResult = oaiResult;
-
+                sb.AppendLine();
+                sb.Append("Recieved a Response with a Status Code of: ");
                 sb.Append(oaiResult.StatusCode);
-            }
-            else
-            {
-                sb.Append(this.RawResult.GetType());
-            }
+                sb.Append(".");
 
-            sb.Append(". Check that your Operation implementation return value matches the OpenAPI operation response definition.");
+                sb.AppendLine();
 
-            this.Message = sb.ToString();
-        }
+                if (oaiResult.Results.Count == 0)
+                {
+                    sb.Append("There were no content result media types provided.");
+                }
+                else
+                {
+                    sb.Append("The following content result media types were provided: ");
+                    appendComma = false;
+                    foreach (string item in oaiResult.Results.Keys)
+                    {
+                        if (appendComma)
+                        {
+                            sb.Append(", ");
+                        }
 
-        /// <inheritdoc cref="Message"/>
-        public new string Message { get; }
+                        appendComma = true;
+                        sb.Append(item);
+                    }
+                }
 
-        /// <summary>
-        /// Gets the OpenApiResult.
-        /// </summary>
-        public OpenApiResult? OpenApiResult { get; }
+                sb.AppendLine();
 
-        /// <summary>
-        /// Gets the Operation.
-        /// </summary>
-        public OpenApiOperation Operation { get; }
+                sb.Append("One of the following content result media types was expected: ");
+                sb.AppendLine();
+                foreach (KeyValuePair<string, OpenApiResponse> item in operation.Responses)
+                {
+                    sb.Append(item.Key);
+                    sb.Append(": ");
 
-        /// <summary>
-        /// Gets the raw Result.
-        /// </summary>
-        public object RawResult { get; }
+                    if (item.Value.Content.Count == 0)
+                    {
+                        sb.Append("No content.");
+                        continue;
+                    }
+
+                    appendComma = false;
+
+                    foreach (string mediaType in item.Value.Content.Keys)
+                    {
+                        if (appendComma)
+                        {
+                            sb.Append(", ");
+                        }
+
+                        sb.Append(mediaType);
+                        appendComma = true;
+                    }
+
+                    sb.AppendLine();
+                }
+            }
+            else
+            {
+                sb.AppendLine();
+                sb.Append("Recieved a Response with a type of ");
```

Comment:
Received

Reply:
This strongly leads me to suspect I didn't type this code in the first place. It all looked really weird and I don't remember writing it.

Maybe I was possessed?